assets/js/socket.js

```diff
@@ -70,9 +70,9 @@ window.liveSocket = liveSocket
 socket.connect()
 
 // Now that you are connected, you can join channels with a topic:
-let channel = socket.channel("topic:subtopic", {})
-channel.join()
-  .receive("ok", resp => { console.log("Joined successfully", resp) })
-  .receive("error", resp => { console.log("Unable to join", resp) })
+// let channel = socket.channel("topic:subtopic", {})
+// channel.join()
+//  .receive("ok", resp => { console.log("Joined successfully", resp) })
+//  .receive("error", resp => { console.log("Unable to join", resp) })
```

Comment:
Why not delete?

Reply:
probably could do... I was feeling generous? 😆

Reply:
removed 👍